src/pybind/mgr/cephadm/serve.py

```diff
@@ -302,16 +297,13 @@ def refresh(host: str) -> None:
                     if match:
                         continue
                 self.log.info(f'Updating {host}:{path}')
-                self._write_remote_file(host, path, content, mode, uid, gid)
+                CephadmServe.loop.run_until_complete(self.ssh._write_remote_file(host, path, content, mode, uid, gid))
```

Comment:
can we create a new method `write_remote_file` without the underscore that looks like so?

```python
class SSHConnection
    def write_remote_file(self, *args):
        return loop.run_until_complete(self._write_remote_file(*args))
```

This way we don't need to call run_until_complete within the serve.py